### PR TITLE
WT-10802 Handle EBUSY error when checkpointing in test/format

### PR DESCRIPTION
**Pull request by etienneptl**
When checkpointing, it is possible to get an `EBUSY` error for various reasons. One of them is when we are trying to drop a checkpoint that is being used.

The current code of test/format assumes checkpoints always succeed. The suggested changes cover the scenario when we are trying to perform a named checkpoint and it fails with the `EBUSY` error.

### CODE REVIEW

**Review by keitharnoldsmith — APPROVED** [review]
LGTM.

I've suggested some changes to the new comment.

File: test/format/checkpoint.c [hunk 1]
```diff
@@ -142,7 +140,13 @@ checkpoint(void *arg)
         else
             trace_msg(session, "Checkpoint #%u start (%s)", ++counter, ckpt_config);
 
-        testutil_check(session->checkpoint(session, ckpt_config));
+        ret = session->checkpoint(session, ckpt_config);
+        /*
+         * It is possible to get EBUSY if we try to use a pre existing checkpoint name that is being
+         * used.
```
[the remark this answers is not in the document]
Comment (sueloverso) [reply test/format/checkpoint.c:146]: I will reply in the ticket - but I'm not sure how format gets into this state.

**Comment by etienneptl** [comment]
Closing the PR for now as my theory might not be correct. We will be working on reproducing the issue and understanding better what is going on before pushing code changes.

**Comment by etienneptl** [comment]
See #9079 for the new version of the fix.

**Pull request closed without merging**